ucsb-cs156-f24/proj-happycows-f24-11
# fix width

## Overview

In this PR, I fix the width of the Profit Table so that there is no empty white space. It is also still responsive to overflow.

## Screenshots (Optional)
![image](https://github.com/user-attachments/assets/f0a34fec-dd9a-4cb2-85a9-d452e15c7f0d)


## Validation (Optional)

1. Download this branch.
2. Run the project.
3. Go to any commons and verify.

## Tests


## Dokku link
https://happycows-blobthebuilder.dokku-11.cs.ucsb.edu/

## Linked Issues

Closes #8
